macros/storage/bucket-policy.mdx

```diff
@@ -2,6 +2,6 @@
 macro: storage-bucket-policy
 ---
 
-A **bucket policy** is a resource-based policy option. It allows users to grant access to buckets in other Scaleway projects and organizations. 
+A **bucket policy** is a resource-based policy option. It allows you to grant more granular access to Object Storage resources.
 
-By default, all Object Storage resources in a project are private and can be accessed only by users of said project. Adding a bucket policy to a bucket allows you to grant access to outside users. You can use different combinations of the policy's component strings to customize your permissions for different purposes as required.
\ No newline at end of file
+By default, all Object Storage resources in a project are private and can be accessed only by users or application with IAM permissions to do so. Adding a bucket policy to a bucket allows you to specify who can perform which actions on a bucket and the objects it contains. You can use different combinations of the policy's component strings to customize your permissions for different purposes as required.
```

Comment:
We could say a "specific/given bucket" to be more accurate and not be confounded with IAM

Reply:
seeing as we're talking about the policy at the bucket level, it's already in a specific bucket, no? I cant find a way to phrase this in a non-redundant way. the fact that I'm talking about "actions" already differentiates one from the other, no?

Reply:
ok